shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
I think I agree with @paulproteus -- while "identity providers" feels more technically correct, I think "login providers" is probably easier for people to grasp quickly. Of course, if user testing suggested otherwise I defer to that.